Eating is now possible on bedrock

francois76 opened the pull request:
Bedrock players are now able to eat (previously, saturation and health was not updated server side).

francois76 committed:
adding eating support on bedrock

HoverEpic commented:
Thank you, I'll check that tomorrow.

shaokeyibb commented:
Wow！Good fix！

HoverEpic commented on proxy/src/main/java/org/dragonet/proxy/network/translator/pe/PEInventoryTransactionPacketTranslator.java:
```diff
@@ -1,181 +0,0 @@
-package org.dragonet.proxy.network.translator.pe;
```
How do you manage to delete a whole file and commit that? Next time, please check all your changes bebore commiting.

HoverEpic reviewed (commented):
I think you should redo this pull request with only necessary changes.

lukeeey commented:
Nah just squash and merge

francois76 commented:
I'm a beginner with git, so I had modified a file by mistake and I didn't wanted it to be commited, so instead of deleting its commit, I have commited its deletion :(
I've squashed my commit and I will do another pull request with one commit ;)

HoverEpic reviewed (approved):
Beautifull!

DefinitlyEvil commented:
cool!
